src/lib/components/modal/MoveCodeSnippet.tsx

```diff
@@ -284,10 +306,13 @@ execute();
                         }}
                       />
                       <Box position="absolute" top={4} right={4}>
-                        <CopyButton
-                          value={item.snippet}
-                          amptrackSection="code_snippet"
-                        />
+                        {/* Workaround for centering Tooltip placement */}
+                        <Flex>
```

Comment:
remove this, and remove `display="inline"` in `CopyTemplate.tsx` instead

also remove comment above